csrc/apis/java/classifier.cpp

```diff
@@ -0,0 +1,128 @@
+// Copyright (c) OpenMMLab. All rights reserved.
+
+#include "classifier.h"
+
+#include "common.h"
+//#include <platform.h>
+//#include <benchmark.h>
+#include <jni.h>
+#include <android/log.h>
+#include <unistd.h>
+#include <stdio.h>
+
+// #include <opencv2/core/core.hpp>
+// #include <opencv2/imgproc/imgproc.hpp>
+
+namespace MMDeployJava {
+
+extern "C" {
+JNIEXPORT jobject JNICALL mmdeployClassifierCreateByPath(JNIEnv* env, jobject thiz,
```

Comment:
As we already use the namespace, it's not necessary to put `mmdeploy` as the prefix of a function name. `mmdeploy` in `MMDeploy::mmdeployClassifierCreatePath` is sort of redundant, isn't it?